controls/views.py

```diff
@@ -2400,3 +2402,114 @@ def poam_export(request, system_id, format='xlsx'):
     else:
         # User does not have permission to this system
         raise Http404
+
+def project_import(request, project_id):
+    """
+    Import an entire project's components and control content
+    """
+    project = Project.objects.get(id=project_id)
+    # Retrieve identified System
+    if request.method == 'POST':
+
+        project_data = request.FILES['import_project_data']
+        file_format = request.POST['file-format']
+        importcheck = False
+        if "importcheck" in request.POST:
+            importcheck = request.POST["importcheck"]
+        if file_format == 'CSV':
+            pass
```

Comment:
The way it is now, the logic might confuse some people (if we don't get back to this code for a while), because it's hardcoded as "JSON" above and then there's a check here for "CSV" that doesn't do anything. I think for now, a TODO is most clear